### PR TITLE
Archive and Delete Page

### PR DESCRIPTION
**Pull request by humzahasan**
PR Description :

PR contain changes for Delete and Archive pages for note web app.

[Only Check Me](https://github.com/humzahasan/notova/pull/6/commits/90f66fe626040f109fef6bde0c0720baefbef788)

[Final Preview Link](https://deploy-preview-10--notova.netlify.app/)

### CODE REVIEW

**Review by srishtigautam21 — APPROVED** [review]
All good
no changes from my side
Good to merge

File: src/pages/Archive/Archive.jsx [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+import { useNote } from "../../context/note-context";
+
+const Archive = () => {
+  const {
+    state: { archives },
+    deleteFromArchive,
+    restoreToNote,
+  } = useNote();
+
+  console.log("archive -", archives);
```
Comment (sudipkundu999) [inline src/pages/Archive/Archive.jsx:13]: Remove console log

**Review by sudipkundu999 — APPROVED** [review]
The code looks clean and understandable.
Good to merge

File: src/pages/Archive/Archive.jsx (hunk 1 above)
Comment (humzahasan) [reply src/pages/Archive/Archive.jsx:13]: Removed in upcoming PR.

**Pull request merged**